src/main/java/uk/gov/cshr/controller/CustomErrorController.java

```diff
@@ -18,11 +17,16 @@ public String handleError(HttpServletRequest request) {
     if (status != null) {
       Integer statusCode = Integer.valueOf(status.toString());
 
-      if (statusCode == HttpStatus.NOT_FOUND.value()) {
-        return "404";
-      }
-      else if (statusCode == HttpStatus.INTERNAL_SERVER_ERROR.value()) {
-        return "500";
+      if (statusCode == HttpStatus.BAD_REQUEST.value()) {
```

Comment:
How about creating a HashMap called "PAGE_URLS" as a constant with page urls and use "statusCode" as a key? Then we could just write:
`return PAGE_URLS.get(statusCode);`